### PR TITLE
feat: support aab in run and deploy

### PR DESCRIPTION
**Pull request by DimitarTachev**
## PR Checklist

- [x] The PR title follows our guidelines: https://github.com/NativeScript/NativeScript/blob/master/CONTRIBUTING.md#commit-messages.
- [x] There is an issue for the bug/feature this PR is for. To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it.
- [x] You have signed the [CLA](http://www.nativescript.org/cla).
- [x] All existing tests are passing: https://github.com/NativeScript/nativescript-cli/blob/master/CONTRIBUTING.md#contribute-to-the-code-base
- [ ] Tests for the changes are included.

## What is the current behavior?
The run and deploy commands are not able to be executed with `--aab`.

## What is the new behavior?
We are able to be run and deploy with `--aab`.

Related to: https://github.com/NativeScript/nativescript-cli/issues/5049

### CODE REVIEW

File: lib/definitions/build.d.ts [hunk 1]
```diff
@@ -17,12 +17,15 @@ interface IiOSBuildData extends IBuildData {
 	iCloudContainerEnvironment: string;
 }
 
-interface IAndroidBuildData extends IBuildData {
+interface IAndroidBuildData extends IBuildData, IAndroidSigningData {
```
Comment (KristianDD) [inline lib/definitions/build.d.ts]: You can try using `IHasAndroidBundle`. There the flag is optional, but you can change it to mandatory and just extend with `Partial<IHasAndroidBundle>` wherever needed.

File: lib/common/mobile/android/android-application-manager.ts [hunk 2]
```diff
@@ -33,13 +36,29 @@ export class AndroidApplicationManager extends ApplicationManagerBase {
...
-	public async installApplication(packageFilePath: string, appIdentifier?: string): Promise<void> {
+	public async installApplication(packageFilePath: string, appIdentifier?: string, buildData?: IAndroidBuildData): Promise<void> {
 		if (appIdentifier) {
 			const deviceRootPath = `${LiveSyncPaths.ANDROID_TMP_DIR_NAME}/${appIdentifier}`;
 			await this.adb.executeShellCommand(["rm", "-rf", deviceRootPath]);
 		}
 
-		return this.adb.executeCommand(["install", "-r", `${packageFilePath}`]);
+		const { dir, name, ext } = parse(packageFilePath);
+		if (ext === ".aab") {
```
Comment (KristianDD) [inline lib/common/mobile/android/android-application-manager.ts]: const.AAB_EXTENSION_NAME

**Comment by DimitarTachev** [comment]
test cli-smoke

**Comment by DimitarTachev** [comment]
test cli-smoke

**Pull request merged**